hash/gameboy.xml

```diff
@@ -24922,17 +24925,21 @@
 		</part>
 	</software>
 
-<!-- Sachen carts -->
+<!-- Sachen multi-carts -->
 
 	<software name="4in1_001">
 		<description>4 in 1 (Euro, 4B-001)</description>
 		<year>19??</year>
 		<publisher>Sachen</publisher>
 		<info name="serial" value="4B-001"/>
 		<part name="cart" interface="gameboy_cart">
+			<feature name="pcb" value="SA-1005A" /> <!-- Also found with SA-111 PCB -->
+			<!-- SA-111 version only have one epoxy blob combining ROM and Sachen MMC1. -->
+			<feature name="u1" value="" /> <!-- ROM -->
+			<feature name="u2" value="" /> <!-- EPOXY BLOB Sachen MMC1 -->
 			<feature name="slot" value="rom_sachen1" />
 			<dataarea name="rom" size="262144">
-				<rom name="4b-001.bin" size="262144" crc="82f06e93" sha1="7506c75fdfa29935afb1e85f5b6013516b2e9f92" offset="0" />
+				<rom name="vol.7.u1" size="262144" crc="82f06e93" sha1="7506c75fdfa29935afb1e85f5b6013516b2e9f92" offset="0" />
```

Comment:
Can you show a source for this change?  What are the board markings?  What's the chip label if any?  I don't want arbitrary renames without sourcing them.  Also please don't add empty features like this - just add the comments if you don't have a value to put in.

Reply:
http://datomatic.no-intro.org/index.php?page=show_record&s=46&n=1604

It's not really that I don't have any input, it's more that the boards don't have any markings. You can see this same style being used in other softlists like snes.xml.

When the board makers are pedantic like Nintendo, you get all kinds of long, meaningful board markings like `U3 16K SRAM`, other times just a `U3` marking. Sachen isn't exactly doing that. Other than a PCB name, not much is printed on the board. But still, it has two slots, one with a ROM and one with an MMC. Isn't that useful information to document?

Reply:
Well, adding a `feature` element with empty attributes is not useful.  You can add a comment containing the information, but the `feature` element should have a well-defined schema with values that could ultimately be machine-parsed to configure emulation.